sinonjs/referee
# Upgrade prettier to 2.x

This PR upgrades `prettier` to latest and removes local configuration, adopting `prettier` default options.

**This breaks compatibility with IE11 and Edge Legacy, and is thus a `MAJOR` version release**

Also:
* upgrade `eslint-config-prettier`
* upgrade `eslint-plugin-prettier`
* run `prettier` on all JavaScript files
    ```sh
    $(npm bin)/prettier -w -u .
    ```

#### Purpose

We are exploring dropping IE11 and other legacy browsers. This PR demonstrates what the first step would look like.

#### Solution  - optional

* Upgrade `prettier`, `eslint-config-prettier` and `eslint-plugin-prettier` to latest
* Remove `.prettierrc`, accepting the default options
* Re-format all JavaScript files using `prettier`
* Configure editors via `.editorconfig`

#### How to verify - mandatory

1. Check out this branch
1. `npm install`
1. `npm run lint`
1. Observe that there are no warnings or errors

#### Checklist for author
